cosmos/sql/tools/java/ycsb/recipes/README.md

```diff
@@ -105,7 +106,20 @@ Following are the most common user mistakes that lead to errors. The error logs
    ```
    The input authorization token can't serve the request. The wrong key is being used….
    ```
-
+4. When reusing VM(s) for previous executions if you see the following error, please delete the VM(s) and re-execute the recipe. 
```

Comment:
Is there any lenient way then VM delete?

Reply:
Removed the FAQ for 'customData' as this will only happen when the branch name for the framework repo is changed.
Users should never see this error if they don't change the branch name parameter value.